spec/features/applications/show_spec.rb

```diff
@@ -0,0 +1,124 @@
+  
+    @application1.pets << @pet_1
+    @application1.pets << @pet_3
+    @application2.pets << @pet_2
+
+  end
+
+  describe 'as a visitor' do
+    describe 'when I visit /applications/:id' do
+      it 'shows applicant information' do
+        #User story 1
+      
+        visit "/applications/#{@application1.id}"
+        expect(page).to have_content(@application1.name)
+        expect(page).to have_content(@application1.street_address)
+        expect(page).to have_content(@application1.city)
+        expect(page).to have_content(@application1.state)
+        expect(page).to have_content(@application1.zip_code)
+        expect(page).to have_content(@application1.description)
+        expect(page).to have_content(@application1.status)
+        expect(page).to have_content(@pet_1.name)
+        expect(page).to have_link(@pet_1.name)
+        expect(page).to_not have_content(@pet_2.name)
+        expect(page).to_not have_link(@pet_2.name)
+      end
+    end
+  end
+
+  describe "adding a pet to an application" do 
+    it "has an add pet section and a search bar to find a pet" do 
+      #US 4
+      visit "/applications/#{@application1.id}"
+      expect(page).to have_content("Add a Pet to this Application")
+      expect(page).to have_field("Enter pet name")
+      expect(page).to have_button("Search")
+    end
+
+    it "searches for pets in the search bar and renders results" do  
+      #US 4
+      visit "/applications/#{@application1.id}"
+      fill_in "Enter pet name", with: "Pom Pom"
+      click_button "Search"
+
+      expect(current_path).to eq("/applications/#{@application1.id}")
+      expect(page).to have_content("Pom Pom")
+      expect(page).to_not have_content("Lobster")
+    end 
+
+    it 'searches for pets with partial name' do
+      #US 8
+      visit "/applications/#{@application1.id}"
+      fill_in "Enter pet name", with: "Pom"
+      click_button "Search"
+
+      expect(page).to have_content("Pom Pom")
+    end
+
+    it 'renders pets with case insensitive partials' do 
+      #US 9 
+      visit "/applications/#{@application1.id}"
+      fill_in "Enter pet name", with: "pOM"
+      click_button "Search"
+
+      expect(page).to have_content("Pom Pom")
+    end
+    
+    it 'adds the pet to under Pets Applied for' do
+      #US 5
+      application1 = Application.create!(name: "Hannah Banana", street_address: "1234 Sugarwood Cir", city: "Newport", state: "Kentucky", zip_code: "41071", description: "I already have a cat and my cat Dave needs a friend. Dave is very friendly and other cat would be a great addition for our household!")
+      visit "/applications/#{application1.id}"
+      fill_in "Enter pet name", with: "Pom Pom"
+      click_button "Search"
+      click_button "Adopt this Pet"
```

Comment:
It looks like your before_each already adds Pom Pom to this application. This test should add a pet that doesn't already have an association to the application (like maybe pet 2?)